### ISSUE TITLE
Some overworld colours are not being changed

### ISSUE THREAD

**Issue by compiling** [bonimy/zelda3-palette-randomizer#4]
These are some screenshots from blackout mode.  Dungeons seem to be working as intended (freezors are invisible!), but there are a lot of things on the overworld that still have some of their normal colours (generally borders).  I'm not sure how much of this is intended behaviour.

![image](https://user-images.githubusercontent.com/8335770/82441295-12af0300-9ae1-11ea-94f2-0c53c62e5257.png)

![image](https://user-images.githubusercontent.com/8335770/82441540-6faab900-9ae1-11ea-938c-04eb2dfc98f6.png)

![image](https://user-images.githubusercontent.com/8335770/82441772-c912e800-9ae1-11ea-8a09-7d660269f80a.png)

![image](https://user-images.githubusercontent.com/8335770/82441867-f3fd3c00-9ae1-11ea-821d-20b25d90413b.png)

**Comment by bonimy** [comment]
The first two pictures are indeed wrong. I was accidentally referencing some palettes twice and I was removing the dupes. I guess I got over-agrressive and removed all of the references for some areas. I'll fix that up.

The border colors were semi-intentional. I did not include them in the rando logic because the same border color can be used in a lot of different areas (trees, rocks, houses, etc), and it would produce very disconnected output to alter them all differently, so I choose to preserve them. However, for blackout mode, I suppose I should include them.
